### PR TITLE
users methods: resetAvatar and createToken

### PR DESCRIPTION
**Pull request by reist**

### CODE REVIEW

File: lib/rocket_chat/messages/user.rb [hunk 1]
```diff
@@ -38,6 +38,22 @@ def create(username, email, name, password, options = {})
...
+      # @return [Boolean]
+      # @raise [HTTPError, StatusError]
+      #
+      def create_token(user_id: nil, username: nil)
+        response = session.request_json(
+          '/api/v1/users.createToken',
+          method: :post,
+          body: user_params(user_id, username),
+        )
+        RocketChat::Token.new response['data']
```
Comment (abrom) [inline lib/rocket_chat/messages/user.rb]: Shouldn't this just be `RocketChat::Token.new response` ? 

https://rocket.chat/docs/developer-guides/rest-api/users/createtoken

File: lib/rocket_chat/messages/user.rb [hunk 2]
```diff
@@ -38,6 +38,22 @@ def create(username, email, name, password, options = {})
       end
 
       #
+      # users.createToken REST API
+      # @param [String] user_id Rocket.Chat user id
+      # @param [String] username Username
+      # @return [Boolean]
```
Comment (abrom) [inline lib/rocket_chat/messages/user.rb]: `# @return [RocketChat::Token]`

File: lib/rocket_chat/messages/user.rb (hunk 1 above)
Comment (reist) [inline lib/rocket_chat/messages/user.rb]: That's a bug in the docs. I should make a pull request there as well...

File: lib/rocket_chat/messages/user.rb (hunk 2 above)
Comment (reist) [inline lib/rocket_chat/messages/user.rb]: Oops, fixed.

**Comment by reist** [comment]
Not sure what to do about the class line length. User doesn't even include all the API calls yet...

**Comment by abrom** [comment]
I'd just disable `Metrics/LineLength` on that class. 

`class User # rubocop:disable Metrics/LineLength`

Could split the class up some more but I think that would make it less readable, not more

**Comment by reist** [comment]
ClassLength, but I agree. Splitting the file along arbitrary limits wouldn't help understanding.

By the way, https://github.com/RocketChat/Rocket.Chat.Docs/pull/324
Hope they merge it quickly.

**Comment by abrom** [comment]
Oops, yes you're right 👍

**Pull request merged**